src/js/components/inputComponent/inputWrapperComponent.js

```diff
@@ -0,0 +1,134 @@
+import Component from "../componentClass.js";
+import InputComponent from "./inputComponent.js";
+import {ERROR} from '../../utils/message.js'
+
+export default class InputWrapperComponent extends Component{
+    constructor(component, store, changeStep){
+        super(component, store, {
+            carStr : '',
+            repeatCnt : -1
+        });
+        this.changeStep = changeStep;
+    }
+
+    init() {
+        this.render();
+    }
+    
+    display(carInputFlag, repeatInputFlag, step) {
+        this.carInput.component.style.display = carInputFlag === true ?  "" : "none";
+        this.repeatInput.component.style.display = repeatInputFlag === true ?  "" : "none";
+
+        if(step === 0) {
+            this.carInput.focus();
+            this.state = {
+                carStr : '',
+                repeatCnt : -1
+            };
+        } else if(step === 1) {
+            this.repeatInput.focus();
+        }
+    }
+    handleNameChange(e){
+        this.state.carStr = this.carInput.component.querySelector("input").value;
+    }
+    handleNameClick(e){
+        if(this.state.carStr.length === 0) {
+            ERROR.NAME_LENGTH();
+            return;
+        }
+        const carList = this.state.carStr.split(" ").join("").split(',');
+        if(carList.findIndex(car => car.length > 5) !== -1) {
+            ERROR.NAME_LENGTH();
+            return;
+        }
+        this.store.carList = carList;
+        this.carInput.disable(true);
+        this.changeStep(1);
+        this.repeatInput.focus();
+    }
+
```

Comment:
```suggestion
        if(!this.isValidCarNames())  return;
        
        this.store.carList = carList;
        this.carInput.disable(true);
        this.changeStep(1);
        this.repeatInput.focus();
    }

    isValidCarNames () {
        const carList = this.state.carStr.split(" ").join("").split(',');
        if(carList.findIndex(car => car.length > 5) !== -1) {
            ERROR.NAME_LENGTH();
            return false;
        }
        
        return true;
    }
```

이런 식으로 함수를 분리하면 가독성이 더 좋아질 수 있지 않을까요?